Conceptpower+Spring/src/integration-test/java/edu/asu/conceptpower/IntegrationTest.java

```diff
@@ -54,26 +45,10 @@ public void setup() throws Exception {
 
         this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
         if (!isSetupDone) {
-            setSecurityContext();
             logger.debug("Started with lucene indexing for integration tests.");
             this.mockMvc.perform(MockMvcRequestBuilders.post("/auth/indexConcepts").principal(principal));
             isSetupDone = true;
             logger.debug("Completed with lucene indexing for integration tests.");
         }
     }
-
-    private void setSecurityContext() {
-        logger.debug("Setting up security context");
-        Collection<ConceptpowerGrantedAuthority> authorities = new ArrayList<>();
-        ConceptpowerGrantedAuthority grantedAuthority = new ConceptpowerGrantedAuthority("admin");
-        grantedAuthority.setAuthority(ConceptpowerGrantedAuthority.ROLE_ADMIN);
-        authorities.add(grantedAuthority);
-
-        AbstractAuthenticationToken auth = new AnonymousAuthenticationToken("admin", principal, authorities);
-        SecurityContext context = new SecurityContextImpl();
-        context.setAuthentication(auth);
-        SecurityContextHolder.setContext(context);
-        logger.debug("Completed setting up security context");
-    }
```

Comment:
for adding a new concept, username and password have to be provided. If that's not the case anymore that's definitely wrong!